### PR TITLE
LPS-149057, LPS-149058, LPS-149367, LPS-148675 | master

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-149057
https://issues.liferay.com/browse/LPS-149058
https://issues.liferay.com/browse/LPS-149367
https://issues.liferay.com/browse/LPS-148675

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMSearch.testcase [hunk 1]
```diff
@@ -49,12 +49,27 @@ definition {
 	}
 
 	@description = "This is a test for LPS-98063. It checks that a user can search for document by synonyms."
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanSearchForDocumentBySynonyms {
+		property test.run.environment = "EE";
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMSearch.testcase:54]: Hey @jaydawu @liuxingye should this only be run on DXP?

Comment (jaydawu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMSearch.testcase:54]: @austinchiang yeah, this feature works on DXP only.

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**